subsystems/LimelightBase.java

```diff
@@ -1,5 +1,55 @@
 package org.frc5587.lib.subsystems;
 
-public abstract class LimelightBase {
-    
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.networktables.*;
+
+public abstract class LimelightBase extends SubsystemBase {
```

Comment:
I really like this!! Just add some more common functions like, `hasTarget`, `setLEDs`, `getHorizontalAngle`,which would be "tv", "ledMode", and "tx"